frontend/package.json

```diff
@@ -31,6 +31,7 @@
     "class-variance-authority": "^0.7.0",
     "clsx": "^2.1.1",
     "cmdk": "^1.0.0",
+    "crypto-js": "^4.2.0",
```

Comment:
?????

Reply:
no co sie kurde dziwisz, musialem to zrobic, bo dotychczas byl sam "crypto" w libce usosa, a jak wiesz on nie dziala na runtime edge nexta, i mi wywalo w konsoli wiec musialem zmienic.